<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapViewWidget</class>
 <widget class="QWidget" name="MapViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="m_pZoom">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="minimum">
    <number>30</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="sliderPosition">
    <number>100</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="m_pView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>581</width>
     <height>391</height>
    </rect>
   </property>
   <property name="dragMode">
    <enum>QGraphicsView::NoDrag</enum>
   </property>
   <property name="viewportUpdateMode">
    <enum>QGraphicsView::FullViewportUpdate</enum>
   </property>
  </widget>
  <widget class="QLabel" name="m_pMapTreeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>zoom:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pMapTreeLabel_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>layer:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_pCurLayer">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pRevert">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>250</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_pShowGrid">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_pMapTreeLabel_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>grid:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
